test/io-tests/configs/no-defaults.config

```diff
@@ -0,0 +1,26 @@
+db-uri = "tmp_db"
+db-schema = "multi,   tenant,setup"
+db-anon-role = "root"
+db-pool = 1
+db-pool-timeout = 100
+db-extra-search-path = "public, test"
+db-channel = "postgrest"
+db-channel-enabled = true
+db-tx-end = "rollback-allow-override"
+db-prepared-statements = false
+server-host = "0.0.0.0"
+server-port = 80
+server-unix-socket = "/tmp/pgrst_io_test.sock"
+server-unix-socket-mode = "777"
+openapi-server-proxy-uri = "https://postgrest.org"
+jwt-secret = "c2VjdXJpdHl0aHJvdWdob2JzY3VyaXR5"
+jwt-aud = "postgrest.org"
```

Comment:
I tried using `https://postgrest.org` here, because I understood https://tools.ietf.org/html/rfc7519#section-4.1.3 (that's what we are linking in the docs) to support either strings or URIs. But this results in an error when starting postgrest, because the `Crypto.JWT` parser for `StringOrUri` tells me, that this is not a valid URI.

How do I add a URI here? If somebody knows how, we could add a hint to the docs. If not for anyone else, it would at least help me :).

Reply:
Weird.. I'm trying the `jwt-aud="https://postgrest.org"` now(on my set-configs branch) and it works.

Reply:
Though there doesn't seem to be any test with an URI.. I've also tried `jwt-aud="//"` and it doesn't fail. Maybe the aud is not being validated at all..

Reply:
I know that it's only treated as URI, if there is a `:` in it. And then it must be valid. I will try again with a valid and an invalid uri.

Reply:
Or at least that's what I thought... I can confirm, any valid and invalid URL works right now.

Reply:
I think we can open a new issue for this one.